src/main/java/core/basesyntax/io/read/ReportReader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.io.read;
+
+import java.io.FileNotFoundException;
+
+public interface ReportReader {
+    public String readReport() throws FileNotFoundException;
```

Comment:
According to the encapsulation principle, you should remove the redundant 'public' modifier from the method signature in the interface, as all methods in an interface are public by default.